server/fleet/software_installer.go

```diff
@@ -35,6 +35,22 @@ func (FailingSoftwareInstallerStore) Exists(ctx context.Context, installerID str
 	return false, errors.New("software installer store not properly configured")
 }
 
+// SoftwareInstallDetailsResult contains all of the information
+// required for a client to pull in and install software from the fleet server
+type SoftwareInstallDetails struct {
+	HostID uint `json:"-" db:"host_id"`
+	// ExecutionID is a unique identifier for this installation
+	ExecutionID string `json:"execution_id" db:"execution_id"`
```

Comment:
just curious: can we get rid of this? or if we need it, can we make it marshal to `install_id`? it's unfortunate but all the public APIs use `install_id` for this field, and it's likely we'll need to rename it in the db